airgun/views/job_invocation.py

```diff
@@ -56,9 +56,7 @@ class target_hosts_and_inputs(WizardStepView):
 
         targetting_type = Select(locator='//div[button[@aria-haspopup="listbox"]]')
         # https://bugzilla.redhat.com/show_bug.cgi?id=2265127
```

Comment:
The bug has been fixed by Dev but yet to be verified by QE so we can verify(using airgun) and remove this bugzilla. Just a non-blocker comment.

Reply:
Same I'm discussing here https://github.com/SatelliteQE/airgun/pull/1464#discussion_r1671249484, we can rebase and merge this afterwards

Reply:
Sure.